recipe/meta.yaml

```diff
@@ -45,10 +47,10 @@ about:
   license_file: LICENSE.txt
```

Comment:
Could you add `license_family: MIT` please?

Reply:
done!